### ISSUE TITLE
Beam search issue

### ISSUE THREAD

**Issue by jackyuanjie1990** [arvind385801/paraphraseGen#8]
Hello,

Is there an issue with the beam_decode function in rvae.py? I think the output of decoder be converted to log_softmax instead of softmax out = F.log_softmax(self.decoder.fc(dec_out)).unsqueeze(0) in sample_decode function.

Thanks,

Jie
